### DISCUSSION TITLE
The homepage displays videos from different users.

### DISCUSSION THREAD

**Discussion by cdorsat** [mediacms-io/mediacms#916] [General]
The "Latest" section on the homepage should randomly display the latest videos uploaded by different users. Because if the same user uploads a large number of videos in a short period of time, other users will lose the opportunity to display. This is a functional experience problem that has been encountered.

**Comment by mgogoulos** [comment]
Not sure I understand what is the issue or problem here  @cdorsat , can you elaborate?

**Comment by cdorsat** [comment]
For example: Mediacms defaults to display the videos in chronological order. Assume that there are four users A, B, C, and D in the system, and then assume that user A uploads a large number of videos (for example, more than 100) in a short period of time. , Users B, C, and D did not upload new videos. Then the homepage will display a large number of videos of user A, and the videos of users B, C, and D will not have a chance to be displayed.

If the four users A, B, C, and D belong to different categories such as: A=Sports, B=News, C=Music, D=Entertainment... then the homepage will display all news-related videos. We hope to improve this experience and diversify it into different categories of films.


For example, the following example shows almost only one user's content!
![p](https://github.com/mediacms-io/mediacms/assets/22068633/770e72a2-588e-46fa-9298-77e6e8707c06)
